app/main.py

```diff
@@ -1,3 +1,79 @@
+from typing import TypeVar
+
+#  "I'm not sure if this is the best solution"
+#  " for creating accurate type annotations, "
+#  "but the others are not working for me."
+
+T = TypeVar("T", bound="Distance")
```

Comment:
this is not a good idea. You can use `Distance`  for type annotation

Reply:
![2023-11-07 22 04 33](https://github.com/mate-academy/py-distance-class/assets/142013825/57d0f466-5b59-412d-a28f-acd05ae985b8)
When I attempt to implement type annotations like this, I encounter an error when running pytest, as indicated in the screenshot.

Reply:
Place `from __future__ import annotations` on the first line, it should resolve the issue. In case not, make sure that you have Python 3.11 installed and bound to the IDE to utilize the latest functionality. You should use the plain `Distance` in the type annotation. If all steps described above are not successful, visit one of the Q&A sessions to resolve the issue.